src/css/variables.less

```diff
@@ -31,6 +31,7 @@
  * ==============
  */
 @base-font-size: 14px;
+@base-line-height: 1.1428571429;
```

Comment:
Should a line height of `16px` with `16px` `padding-top/bottom` deliver us a 48px height in total, where line-height = `@base-spacing-unit*2` ?